conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -63,10 +63,13 @@ jobs:
         call activate base
         {{ build_setup.replace("\n", "\n        ").rstrip() }}
       displayName: conda-forge build setup
-    {% endif %}
+    {%- endif %}
 
     - script: |
         call activate base
+        if EXIST LICENSE.txt (
+          copy LICENSE.txt "{{ recipe_dir }}\\recipe-scripts-license.txt"
+        )
```

Comment:
Would it make sense to name this `FEEDSTOCK_LICENSE.txt` or similar? Also wdyt about always using that name?

Reply:
Ping on this @jakirkham

Reply:
Sorry for the slow reply 😞

Guess the question is, how do we handle the case of the recipe having a license of its own ( https://github.com/conda-forge/conda-smithy/issues/229 )?

Reply:
Indeed though do we also want the feedstock license in that case?

Reply:
Ah. In that case, they are already copied because they are in the recipe directory.

> Also will this clobber the license that is already there?

Unless the name is `recipe-scripts-license.txt` it wouldn't.

Reply:
Good point. This seems good then

Reply:
Thanks